aldryn_forms/cms_plugins.py

```diff
@@ -208,18 +234,14 @@ def serialize_value(self, instance, value, is_confirmation=False):
             value = '-'
         return unicode(value)
 
-    def serialize_field(self, form, instance, is_confirmation=False):
-        """Returns a (label, value) tuple for the given field.
-
-        Both fields will have been converted to a string object."""
-        key = self.get_field_name(instance)
-        value = self.serialize_value(instance, form.cleaned_data[key],
-                                     is_confirmation)
-        name = instance.label or instance.placeholder_text or key
-        return name, value
-
-    def get_form_fields(self, instance):
-        return {self.get_field_name(instance=instance): self.get_form_field(instance=instance)}
+    def serialize_field(self, form, field, is_confirmation=False):
+        """Returns a (key, label, value) named tuple for the given field."""
+        value = self.serialize_value(
+            instance=field.plugin_instance,
+            value=form.cleaned_data[field.name],
+            is_confirmation=is_confirmation
+        )
+        return SerializedFormField(name=field.name, label=field.label, value=value)
```

Comment:
this method gets called quite a lot, therefore avoiding a ```module.``` lookup is always encouraged but instead of defining it in module scope I just imported it as suggested by tom.